data-serving/data-service/src/model/travel.ts

```diff
@@ -0,0 +1,32 @@
+import { DateRangeDocument, dateRangeSchema } from './date-range';
+import { LocationDocument, locationSchema } from './location';
+
+import mongoose from 'mongoose';
+
+export enum TravelPurpose {
+    Family = 'Family',
+    Conference = 'Conference',
+    Work = 'Work',
+}
+
+export const travelSchema = new mongoose.Schema(
+    {
+        location: locationSchema,
+        dateRange: dateRangeSchema,
+        purpose: {
+            type: String,
+            enum: Object.values(TravelPurpose),
+        },
+        additionalInformation: String,
+    },
+    {
+        strict: true,
```

Comment:
Do we have this specified `false` or `throw` at some top-level point? `true` is the default value

Reply:
nope -- removed (here & elsewhere)